OracleDatabase/RAC/OracleConnectionManager/dockerfiles/21.3.0/Dockerfile

```diff
@@ -0,0 +1,85 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2019 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Connection Manager
+# 
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) LINUX.X64_213000_client.zip 
+#     Oracle Database 21c Client (21.3) for Linux x86-64
+#     Download Oracle Connection Manager as part of the Oracle Database Client
+#     from https://www.oracle.com/technetwork/database/enterprise-edition/downloads/oracle21c-linux-210000-5022980.html 
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run: 
+#      $ docker build -t oracle/cman:21.3.0 . 
+#
+# Pull base image
+# ---------------
+FROM oraclelinux:7-slim
+
+# Maintainer
+# ----------
+MAINTAINER Sanjay Singh (Sanjay.Singh@oracle.com), Paramdeep Saini (paramdeep.saini@oracle.com)
```

Comment:
The `MAINTAINER` tag is deprecated. Please see the contribution guide for the required and recommended labels to use instead.

Reply:
Removed MAINTAINER clause.